android/anc/src/main/java/org/smartregister/fhircore/anc/ui/anccare/register/AncItemMapper.kt

```diff
@@ -31,7 +30,7 @@ object AncItemMapper : DomainMapper<Patient, AncPatientItem> {
 
   override fun mapToDomainModel(dto: Patient): AncPatientItem {
     val name = dto.extractName()
-    val gender = dto.extractGender(AncApplication.getContext()).first()
+    val gender = dto.extractGender().first()
```

Comment:
Removing the context will not allow for translation.

Reply:
context is unused field. I did not find the translation part in engine